paddle/fluid/operators/mkldnn/conv_mkldnn_op.cc

```diff
@@ -602,12 +601,12 @@ class ConvMKLDNNOpKernel : public paddle::framework::OpKernel<T> {
           platform::SetDstMemoryHandler<uint8_t>(ctx, output, handler,
                                                  &dst_memory_p);
         } else {
-          need_s8_to_u8 = fuse_relu;
+          need_s8_to_u8 = fuse_relu || fuse_brelu;
           platform::SetDstMemoryHandler<int8_t>(ctx, output, handler,
                                                 &dst_memory_p);
         }
       } else if (!force_fp32_output) {
-        if (fuse_relu) {
+        if (fuse_relu || fuse_brelu) {
```

Comment:
How about using a variable to compute `fuse_relu || fuse_brelu` at first?
`bool xxx = fuse_relu || fuse_brelu`

Reply:
> How about using a variable to compute `fuse_relu || fuse_brelu` at first?
> `bool xxx = fuse_relu || fuse_brelu`

Yes. I agree with you. I will change. It is about using uint8_t or int8_t